safe/gui/tools/print_report_dialog.py

```diff
@@ -233,15 +234,21 @@ def print_as_pdf(self):
         report_path = os.path.dirname(impact_layer.source())
 
         # Get the hazard and exposure definition used in current IF
-        hazard = definition(
-            QgsExpressionContextUtils.projectScope().variable(
-                'hazard_keywords__hazard'))
-        exposure = definition(
-            QgsExpressionContextUtils.projectScope().variable(
-                'exposure_keywords__exposure'))
+        hazard = QgsExpressionContextUtils.projectScope().variable(
```

Comment:
Sorry, I just realized.  We want to clean these variables in the project scope.
The end user can't use them out of the box so IMHO it's not good to expose them to public. Can you fetch these keywords with the impact layer?

Reply:
yes of course